app/src/main/java/app/gamenative/service/SteamAutoCloud.kt

```
@@ -1078,6 +1082,40 @@ object SteamAutoCloud {
             microsecUploadFiles = microsecUploadFiles,
         )
 
+        // Write remotecache.vdf after successful sync
+        if (syncResult == SyncResult.Success || syncResult == SyncResult.UpToDate) {
+            try {
+                // Calculate total size and SHA of all synced files
+                val allFiles = getLocalUserFilesAsPrefixMap().values.flatten()
+                val totalSize = allFiles.sumOf {
+                    try {
+                        Files.size(it.getAbsPath(prefixToPath))
+                    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)